tests/conftest.py

```diff
@@ -155,3 +156,14 @@ def lang_data_many(lang_type, lic_type, lang_data, service, identity):
         service.create(identity, data)
     Vocabulary.index.refresh()  # Refresh the index
     return lang_ids
+
+
+# FIXME: https://github.com/inveniosoftware/pytest-invenio/issues/30
+# NOTE: Without this, success of test depends on test order
+@pytest.fixture()
+def cache():
```

Comment:
Nice catch! 🚀 Do you know if we still need the fixture in `invenio-records-resources` or it could also be removed from there?

Reply:
I'd take a cautious approach. If cache is only required for read_all/read_many then yes it could be removed. I don't have the latest code of invenio-record-resources (I was debugging cache functionality :grin: from the old code)